docker/cloudbuild.yaml

```diff
@@ -47,7 +46,6 @@ substitutions:
     _CUDA_COMPUTE: '3.7,7.0,7.5,8.0'
     _GITHUB_XLA_BRANCH: ''
     _GITHUB_EXAMPLE_BRANCH: 'master'
-    _BUILD_CPP_TESTS: '1'
```

Comment:
Do we now have another way to turn on/off the CPP test build? If so, we need to expose it to GCB, since in some environments the test build fails... having said that, have we tested the new bazel build with Python3.9 (both with and without CPP tests)?

Reply:
See above.